gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/DuplicateObservationStatementMapper.java

```diff
@@ -0,0 +1,178 @@
+                truncatedObservationText.length() - CHAR_LIMIT_FOR_TRUNCATION,
+                truncatedObservationText.length() - ELLIPSES.length()
+        );
+        return candidateMatchingPertinentAnnotation.getText().contains(truncatedObservationTextWithoutEllipses);
+    }
+
+    private static boolean isAnnotationTruncated(RCMRMT030101UKObservationStatement observationStatement) {
+        String annotationText = getPertinentAnnotation(observationStatement).getText();
+        return annotationText.endsWith(ELLIPSES) && annotationText.length() >= CHAR_LIMIT_FOR_TRUNCATION;
+    }
+
+    private static boolean isMergeAllowedLinkedStatement(RCMRMT030101UKObservationStatement obs) {
+        return (obs.getPriorityCode() == null
+                && obs.getUncertaintyCode() == null
+                && obs.getValue() == null
+                && obs.getInterpretationCode() == null
+                && obs.getSubject() == null
+                && (obs.getSpecimen() == null || obs.getSpecimen().isEmpty())
+                && (obs.getReferenceRange() == null || obs.getReferenceRange().isEmpty())
+                && (obs.getInformant() == null || obs.getInformant().isEmpty())
+                && (obs.getParticipant() == null || obs.getParticipant().isEmpty())
+                && (obs.getReplacementOf() == null || obs.getReplacementOf().isEmpty())
+                && (obs.getReason() == null || obs.getReason().isEmpty())
+                && (obs.getReference() == null || obs.getReference().isEmpty())
+                && (obs.getSequelTo() == null || obs.getSequelTo().isEmpty()));
+    }
+
+    private static boolean isMergeAllowedtatement(RCMRMT030101UKObservationStatement observationStatement) {
+        return  observationStatement.getPriorityCode() == null
+                && observationStatement.getUncertaintyCode() == null
+                && observationStatement.getValue() == null
+                && observationStatement.getInterpretationCode() == null
+                && observationStatement.getSubject() == null
+                && (observationStatement.getSpecimen() == null || observationStatement.getSpecimen().isEmpty())
+                && (observationStatement.getReferenceRange() == null || observationStatement.getReferenceRange().isEmpty())
+                && (observationStatement.getInformant() == null || observationStatement.getInformant().isEmpty())
+                && (observationStatement.getParticipant() == null || observationStatement.getParticipant().isEmpty())
+                && (observationStatement.getReplacementOf() == null || observationStatement.getReplacementOf().isEmpty())
+                && (observationStatement.getReason() == null || observationStatement.getReason().isEmpty())
+                && (observationStatement.getReference() == null || observationStatement.getReference().isEmpty())
+                && (observationStatement.getSequelTo() == null || observationStatement.getSequelTo().isEmpty());
+    }
```

Comment:
@azam145 These methods are identical?! Why are there two methods for the same thing?